test/scenarios/move_overwriting_dir/not_trashed/scenario.js

```diff
@@ -46,24 +44,36 @@ module.exports = ({
       'dst/dir/subdir/subsub/',
       'src/'
     ],
-    trash:
-      process.platform !== 'darwin' && runWithStoppedClient()
+    localTrash: [
+      // Why isn't the hierarchy preserved in the local trash on Windows?
```

Comment:
Is it something you want to fix in this PR?

Reply:
Probably not. I guess it's related to the fact that remote changes are fetched together and handled differently from the Windows local changes way.
Since the result is the same on all platforms and other platforms get this result when propagating local changes this does not seem too important to fix now.